File: src/test/java/no/nav/permitteringsskjemaapi/ApplicationTests.java
package no.nav.permitteringsskjemaapi;

import org.junit.jupiter.api.Test;
import org.springframework.boot.test.context.SpringBootTest;
import org.springframework.context.annotation.Profile;
import org.springframework.test.context.ActiveProfiles;
import org.springframework.test.context.TestPropertySource;

@SpringBootTest
@ActiveProfiles("test")
class ApplicationTests {

    @Test
    void contextLoads() {
    }

}

File: src/test/kotlin/no/nav/permitteringsskjemaapi/ApplicationTests.kt
package no.nav.permitteringsskjemaapi

import org.junit.jupiter.api.Test
import org.junit.jupiter.api.extension.ExtendWith
import org.springframework.boot.test.context.SpringBootTest
import org.springframework.test.context.ActiveProfiles
import org.springframework.test.context.junit.jupiter.SpringExtension


@SpringBootTest
@ExtendWith(SpringExtension::class)
@ActiveProfiles("test")
internal class ApplicationTests {
    @Test
    fun contextLoads() {
    }
}